gemrb/core/Map.h

```diff
@@ -627,8 +627,8 @@ class GEM_EXPORT Map : public Scriptable {
 	PathListNode* GetLine(const Point &start, int Steps, orient_t Orientation, int flags) const;
 	PathListNode* GetLine(const Point &start, const Point &dest, int speed, orient_t Orientation, int flags) const;
 	Path GetLinePath(const Point &start, const Point &dest, int speed, orient_t Orientation, int flags) const;
-	/* Finds the path which leads to near d */
-	PathListNode* FindPath(const Point &s, const Point &d, unsigned int size, unsigned int minDistance = 0, int flags = PF_SIGHT, const Actor *caller = NULL) const;
+	/* Finds the path which leads to near d. Return path_length if provided. */
+	PathListNode* FindPath(const Point &s, const Point &d, unsigned int size, unsigned int minDistance = 0, int flags = PF_SIGHT, const Actor *caller = NULL, int *path_length = NULL) const;
```

Comment:
```suggestion
	PathListNode* FindPath(const Point& s, const Point& d, unsigned int size, unsigned int minDistance = 0, int flags = PF_SIGHT, const Actor* caller = nullptr, int* pathLength = nullptr) const;
```
Please use nullptr with new code, same for camel case and symbol placement. We're slowly unifying the styles.